src/modules/vtol_att_control/standard.cpp

```diff
@@ -409,7 +419,8 @@ void Standard::fill_actuator_outputs()
 		fw_out[actuator_controls_s::INDEX_PITCH]        = fw_in[actuator_controls_s::INDEX_PITCH];
 		fw_out[actuator_controls_s::INDEX_YAW]          = fw_in[actuator_controls_s::INDEX_YAW];
 		fw_out[actuator_controls_s::INDEX_THROTTLE]     = fw_in[actuator_controls_s::INDEX_THROTTLE];
-		fw_out[actuator_controls_s::INDEX_FLAPS]        = fw_in[actuator_controls_s::INDEX_FLAPS];
+		fw_out[actuator_controls_s::INDEX_FLAPS]        = _flaps_setpoint_with_slewrate.getState();
```

Comment:
@sfuhrer In FW mode shouldn't you just feedforward the actuator controls from the FW module?

Reply:
But then we'd possibly have a step in the controls when the vehicle switches from a transition to FW and the other way around. What do we lose by always going through the slew rate in the VTOL module like proposed here?